### PR TITLE
[FIX] core: correctly handle major-less upgrade scripts during major version change

### PR DESCRIPTION
**Pull request by KangOl**
…

### CODE REVIEW

File: odoo/modules/migration.py [hunk 1]
```diff
@@ -117,8 +118,11 @@ def migrate_module(self, pkg, stage):
             return
 
         def convert_version(version):
-            if version.count('.') >= 2:
-                return version  # the version number already containt the server version
+            # determine if the version starts with an odoo version.
+            saas = ("saas~" in version)
+            prefixed = re.match(r"[1-9]+\.0\.", version)
```
Comment (aj-fuentes) [inline odoo/modules/migration.py:123]:
```suggestion
            prefixed = re.match(r"^[1-9]+\.0\.", version)
```

**Comment by KangOl** [comment]
@robodoo r+

**Comment by d-fence** [comment]
robodoo retry
forgot to remove ci/l10n for 11.0 and its little brothers.

**Comment by d-fence** [comment]
robodoo retry
phantomjs failure ...

**Pull request closed without merging**